copregion/closedsource/docker-compose.yml

```diff
@@ -363,6 +364,7 @@ services:
          - EMAIL_PASSWORD=${REGION_EMAIL_PASSWORD}
          - EMAIL_PORT=${REGION_EMAIL_PORT}
          - EMAIL_USER=${REGION_EMAIL_USER}
+         - MGMT_IP=${ROUTER_MANAGEMENT_IP}
```

Comment:
Why did you add back the `MGMT_IP` that I renamed this morning to `ROUTER_MGMT_IP` (line 378)?

In robot setting file it is `MGMT_IP = os.getenv('ROUTER_MGMT_IP', '')`, which means it will set `MGMT_IP` in the settings file to the variable `ROUTER_MGMT_IP` if passed in during docker build stage.

Can you please remove this again int he files you have added it.

Reply:
that was by mistake while correcting merge conflicts